### PR TITLE
N25 final

### PR DESCRIPTION
**Pull request by AngelOnFira**
_Part of #751_

Preview (f628ccb): https://agitated-saha-af60ca.netlify.app/news/025/

Todo:

- [x] Add sections at the bottom
- [x] Line length rewrap
- [x] Full spelling and grammar pass

~~Aiming to release this in around 12 hours.~~
Expected release time is 18:00 GMT

### CODE REVIEW

**Comment by septum** [comment]
Hi! There's a slight mistake with the developers of TO BE DIRE. This was addressed in the Rusty Jam's Discord and the game's itch.io page. So, the change would be in the `index.md` in line 73  :
```
- Third place: [To be Dire][rusty-jam-third-place] by septum
```
To be changed with:
```
- Third place: [TO BE DIRE][rusty-jam-third-place] by mdaffin, 
  TimeLark and septum
```

**Review by kvark — APPROVED** [review]

File: content/news/025/index.md [hunk 1]
```diff
@@ -599,8 +592,8 @@ aims to be easy to use and provide a large set of out-of-the-box features.
 [![rg3d feature highlights](rg3d-feature-highlights.png)][rg3d_fh]
 __Engine feature highlights__
 
-Check out the video about the engine made by [@GameFromScratch][gfs_channel],
-an article could be found [here][gfs_article].
+Check out the video about the engine made by [@GameFromScratch][gfs_channel], an
```
Comment (ozkriff) [inline content/news/025/index.md:595]: There're three images in rg3d's section - the logo and two previews for videos - while the guideline is one image per section.

I propose removing the logo and replacing the GFS's video with just a text link.

Also, the list is formatted as a source code - this gors against the style guide as it'll cause horizontal scroll on mobile.

**Review by ozkriff — COMMENTED** [review]
LGTM overall, though I've marked a couple of style issues in comments where possible.

Also the rx section violates a few style guidelines.
  First, the template assumes that the website, code, and community links should be placed into a braced list like "rx (website, code, Discord) by @cloudhead is a modern and minimalist pixel editor written in Rust."
  Second, the changelist is formatted as code block and it'll cause horizontal scroll on mobile.

File: content/news/025/index.md [hunk 2]
```diff
@@ -496,8 +490,8 @@ has been announced. The NFT will only be acquirable for a limited amount of time
 
 [As promised](https://fishfight.itch.io/ff/devlog/281554/fish-fight-reloaded),
 after months of private prototyping of early invitees, the Fish Fight devs are
-finally ready to make their code and community channels available to the
-general public. They also launched their website!
+finally ready to make their code and community channels available to the general
```
Comment (ozkriff) [inline content/news/025/index.md:493]: FishFight section lacks a one-sentence description of the game. Some of the links - GitHub repo, Discord server, and fishfight.org - should be moved into a braced list there.

**Comment by AngelOnFira** [comment]
@ozkriff All of your notes should be resolved :100:

**Review by ozkriff — APPROVED** [review]

**Review by 17cupsofcoffee — APPROVED** [review]

**Comment by AngelOnFira** [comment]
I'll aim to merge in about 2 hours, which will hopefully give any final reviewers time to check out stuff.

**Pull request merged**